omx/gstomxvideodec.c

```diff
@@ -1290,33 +1290,30 @@ gst_omx_video_dec_loop (GstOMXVideoDec * self)
   GstOMXAcquireBufferReturn acq_return;
   GstClockTimeDiff deadline;
   OMX_ERRORTYPE err;
-  gboolean flush_flag;
 
 #if defined (USE_OMX_TARGET_RPI) && defined (HAVE_GST_GL)
   port = self->eglimage ? self->egl_out_port : self->dec_out_port;
 #else
   port = self->dec_out_port;
 #endif
 
-  GST_VIDEO_DECODER_STREAM_LOCK (self);
-  flush_flag = self->flush_flag;
-  GST_VIDEO_DECODER_STREAM_UNLOCK (self);
-
-  if (flush_flag) {
+  if (self->flush_flag) {
```

Comment:
keep flush flag protection

Reply:
Added in 649bdbaf446bccb9760859a2c0f8ea3e5ad6f03c